sympy/polys/ring_series.py

```diff
@@ -572,6 +574,111 @@ def _coefficient_t(p, t):
             p1[monomial_div(expv, expv1)] = p[expv]
     return p1
 
+def rs_fast_series_reversion(p,x,n):
```

Comment:
I suggest naming this after the algorithm e.g. `rs_series_reversion_newton` or something. If the old function is to be kept then it should also be renamed so that there can be a new function called `rs_series_reversion` that calls whichever of these two functions is appropriate.